Makefile

```diff
@@ -228,9 +232,9 @@ dist/binaries:
     		for os_arch in ${TARGETS}; do \
     			goos=`echo $$os_arch|sed 's|/.*||'`; \
     			goarch=`echo $$os_arch|sed 's|^.*/||'`; \
-    		    GOOS=$$goos GOARCH=$$goarch $(GO) build -tags webconsole -v -ldflags '${V_LDFLAGS_COMMON}' -o ./dist/$$service-v${VERSION}-$$goos-$$goarch ./cmd/$$service/$$service.go ; \
+				GOOS=$$goos GOARCH=$$goarch V_BUILD_NAME=./dist/$$service-v${VERSION}-$$goos-$$goarch make $$service ; \
```

Comment:
How about some computed variable names? https://www.gnu.org/software/make/manual/html_node/Computed-Names.html

That way we could have some variable like `V_BUILD_FLAGS_immudb` and access it with `$(V_BUILD_FLAGS_$$service)`?

Reply:
Thanks for the suggestion! I will try this out